### PR TITLE
feat(components): update toast visuals and timing [JOB-27572]

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

The Toast, while intended as a quick, non-critical message, is a bit too unobtrusive; admins are missing the success messages when they save changes, for example.

This is particularly problematic on larger/multiple monitor setups, which we know many of our admins are operating on.

### Changed

- Toast now uses the `reverse` colors from the theme for high contrast against the main application surface
- Toast now has longer min and max durations, and a longer auto-calculated duration

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Toast/Toast.tsx [hunk 1]
```diff
@@ -102,9 +102,9 @@ export function Toast({
   }
 
   function getTimeout() {
-    const min = 2000;
-    const max = 5000;
-    const time = message.length * 100;
+    const min = 5000;
+    const max = 10000;
+    const time = message.length * 200;
```
Comment (adambobadam) [inline packages/components/src/Toast/Toast.tsx:107]: Perhaps too much of a smartboi comment (sorry!) figure I'll ask anyways.  Should time here also account for something like the variation type as well?

**Review by adambobadam — APPROVED** [review]

File: packages/components/src/Toast/Toast.tsx (hunk 1 above)
Comment (chris-at-jobber) [reply packages/components/src/Toast/Toast.tsx:107]: If the reasoning is like "stick around longer for error"...

TLDR we should actually remove error as an available Toast type and only use Banner for errors (but that's a separate issue, imo)

Comment (adambobadam) [reply packages/components/src/Toast/Toast.tsx:107]: Okay gotcha!  Ya, not even an issue here, just figured I'd ask if something like `info` should stick around for slightly longer than `success`/`default`
